lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -2377,7 +2377,7 @@ def option_values_target_addrs
 
     # List all hosts in the database
     else
-      framework.db.each_host(framework.db.workspace) do |host|
+      framework.db.hosts.each do |host|
```

Comment:
This will work because the data proxy adds the workspace to the request for `hosts`. However, consider explicitly specifying the workspace as a best practice.
```
framework.db.hosts(workspace: framework.db.workspace).each
```